pytorch_lightning/core/datamodule.py

```diff
@@ -20,9 +20,9 @@
 from torch.utils.data import DataLoader, Dataset, IterableDataset
 
 from pytorch_lightning.core.hooks import CheckpointHooks, DataHooks
+from pytorch_lightning.core.mixin import HyperparametersMixin
```

Comment:
Most directories are plural

```suggestion
from pytorch_lightning.core.mixins import HyperparametersMixin
```

Reply:
Updated!